luigi/execution_summary.py

```diff
@@ -126,15 +133,15 @@ def _get_str(task_dict, extra_indent):
                 attributes = sorted(params[unique_param])
                 if _ranging_attributes(attributes, unique_param) and len(attributes) > 2:
                     ranging = True
-                    row += '- {0} {1}({2}'.format(len(tasks), task_family, _get_str_ranging_multiple_parameters(attributes, tasks, unique_param))
+                    line = prefix + '- {0} {1}({2}'.format(len(tasks), task_family, _get_str_ranging_multiple_parameters(attributes, tasks, unique_param))
             if not ranging:
                 if len(tasks) == 1:
-                    row += '- {0} {1}'.format(len(tasks), tasks[0])
+                    line = prefix + '- {0} {1}'.format(len(tasks), tasks[0])
                 if len(tasks) == 2:
-                    row += '- {0} and {1}'.format(tasks[0], tasks[1])
+                    line = prefix + '- {0} {1} and {2}'.format(len(tasks), tasks[0], tasks[1])
                 if len(tasks) > 2:
-                    row += '- {0} and {1} other {2}'.format(tasks[0], len(tasks) - 1, task_family)
-        lines.append(row)
+                    line = prefix + '- {0} {1} ...'.format(len(tasks), tasks[0])
```

Comment:
Hmm, Wasn't it nicer and shorter to just do `row += blah` instead of `line =  prefix + blah`. I don't mind the new variable name, but I think it looked ok with `+=`, why did you remove that?